### PR TITLE
Align "workgroup" spelling

### LINKED ISSUE #89: Align use of "workgroup" spelling across website
Currently, we use both "work group" (with space) and "workgroup".

We decided to align this across the website to only use the latter spelling, without a space. This is also the recommended spelling by Apple Style Guide.

### PR DESCRIPTION
**Pull request by alexandersandberg**
This fixes #89.

### Motivation:

As discussed among SWWG, we decided that it would be a good idea to align the spelling of "workgroup" according to the [Apple Style Guide](https://help.apple.com/applestyleguide/). This is the first step in a plan to introduce a style guide for Swift.org content.

### Modifications:

- Change any occurrences of "work group(s)" to "workgroup(s)"...
  - in static website content
  - in two recent blog posts
  - in `_data/authors.yml` for Diversity in Swift workgroup

### Discussions:

**Should we also update the spelling of "workgroup" in older blog posts?**

**Should we also update the spelling of "workgroup" in personal author bios in `_data/authors.yml`?**

### CODE REVIEW

**Comment by alexandersandberg** [comment]
@swift-ci test

**Review by tomerd — APPROVED** [review]

**Review by 0xTim — APPROVED** [review]

**Comment by tomerd** [comment]
cc @tkremenek

File: diversity/index.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The current Diversity in Swift work group consists of the following people:
 * Paul Hudson ([@paul_hudson](https://forums.swift.org/u/paul_hudson/))
 * Timirah James ([@timirahj](https://forums.swift.org/u/timirahj))
 
-You can reach the Diversity in Swift work group on the Swift Forums as [@diversity-work-group](https://forums.swift.org/g/diversity-work-group).
+You can reach the Diversity in Swift workgroup on the Swift Forums as [@diversity-work-group](https://forums.swift.org/g/diversity-work-group).
```
Comment (shahmishal) [inline diversity/index.md:25]:
```suggestion
You can reach the Diversity in Swift workgroup on the Swift Forums as [@diversity-work-group](https://forums.swift.org/g/diversity-workgroup).
```

Last week I updated forums.swift.org to workgroup from work-group.

File: _data/authors.yml [hunk 2]
```diff
@@ -259,8 +259,8 @@ parkera:
    about: "Tony Parker manages teams at Apple working on Foundation and Swift packages."
 
 diversity-work-group:
```
Comment (shahmishal) [inline _data/authors.yml:261]:
```suggestion
diversity-workgroup:
```
We should also update the blog post which contains this reference

**Review by shahmishal — CHANGES_REQUESTED** [review]

File: _data/authors.yml (hunk 2 above)
Comment (shahmishal) [reply _data/authors.yml:261]: - _posts/2020-12-16-accessibility-and-inclusion.md
- _posts/2020-12-16-diversity-in-swift.md
- _posts/2021-05-10-swift-mentorship-program.md

File: diversity/index.md (hunk 1 above)
Comment (shahmishal) [reply diversity/index.md:25]: Also, we should update line 31.

**Comment by shahmishal** [comment]
Few more files we should update the text: 
- gsoc2022/index.md
   - Line 397
- mentorship-tos/index.md
   - Line 32
- mentorship/index.md 
   - Line 38, 98 and 104 


@alexandersandberg Thank you for getting the website updated to "workgroup"!

File: diversity/index.md (hunk 1 above)
Comment (alexandersandberg) [reply diversity/index.md:25]: Oh, I was not aware of this change. Thanks!

https://forums.swift.org/g/diversity-workgroup seems to be private?

**Comment by alexandersandberg** [comment]
Thank you, @shahmishal!

I have updated the diversity workgroup and interop workgroup references in e83f00a and 7db3888.

I also added a link to one of the diversity workgroup references in c382fcd.

**Comment by shahmishal** [comment]
> Should we also update the spelling of "workgroup" in older blog posts?

It would be good to figure out how many blog posts we are talking about.

> Should we also update the spelling of "workgroup" in personal author bios in _data/authors.yml?

We should and `cc` the authors make them aware of the change.

**Comment by alexandersandberg** [comment]
@shahmishal 

> It would be good to figure out how many blog posts we are talking about.

Only 3 blog posts would need to be updated:
- `_posts/2016-10-25-server-api-workgroup.md` (not sure if we need to update this as this workgroup no longer exists?)
- `_posts/2019-10-31-sswg-update-2019.md`
- `_posts/2021-03-24-womens-history-month.md`

However, as we may not enforce the planned style guide for blog posts, I would only update references to workgroups (e.g. "Swift Server Work Group"), not the use of the noun "work group" (e.g. on [this line](https://github.com/apple/swift-org-website/blob/327234d321faa7db01f3d6e7e1edcc80369298a1/_posts/2019-10-31-sswg-update-2019.md?plain=1#L142)). Does this make sense?

> We should and `cc` the authors make them aware of the change.

Good idea. 👍 I will do this.

**Comment by alexandersandberg** [comment]
Hey @twostraws, @tanner0101, @tomerd, @0xTim, and @devanshimodha, I just wanted to let you know that we have made some minor updates to your author bios in this PR to align the spelling of "workgroup" across the website (more info in this PR).

See the changes [here](https://github.com/apple/swift-org-website/pull/90/files#diff-fdca1c48b249c38bee14489176341e078b9cfcf168e19ba1c669b92fa7322c49).

No action is required on your part, but please let me know if you have any concerns. Thank you! 🙂

**Comment by twostraws** [comment]
No concerns – thank you! 🙌

**Review by kaishin — APPROVED** [review]
Much nicer! Thank you.

**Comment by tomerd** [comment]
@swift-ci test

**Review by shahmishal — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**